src/utils/email.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+
+import { URL } from 'url'
+
+import AppConstants from '../app-constants.js'
+
+import { createTransport } from 'nodemailer'
+import mozlog from './log.js'
+
+// TODO
+// import { getTemplate, verifyPartial } from '../views/email-2022.js'
+
+const log = mozlog('email-utils')
```

Comment:
Is this still preferred over `console.*`? I noticed there's a `log.js` in `./utils` so I assumed so, but I see we're using the former in some places in `./src`.